public/app/features/templating/adhoc_variable.ts

```diff
@@ -49,7 +49,7 @@ export class AdhocVariable implements Variable {
       return {
         key: values[0],
         operator: values[1],
-        value: values[2],
+        value: values.slice(2).join('|'),
```

Comment:
How do you escape/unescape delimiter `|` with compatibility?
If we use encode/decodeURIComponent, `|` becomes `%7C`.
But if someone is using ad-hoc variable value=`%7C`, it will be decoded to `|` unexpectedly.
Is this OK?